pkg/kubelet/container/runtime.go

```diff
@@ -52,7 +52,7 @@ type Version interface {
 // information about the different image types.
 type ImageSpec struct {
 	Image string
-	Pod *v1.Pod
+	Pod   *v1.Pod
```

Comment:
There is an extra space?

Reply:
this is automatically updated by "make update"